packages/paste-design-tokens/CHANGELOG.md

```diff
@@ -5,192 +5,94 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 # 2.3.0 (2019-08-15)
 
-
 ### Bug Fixes
 
-* **design-tokens:** resolve path correctly ([6f89111](https://github.com/twilio-labs/paste/commit/6f89111))
-* **tokens:** added missing SG token values ([#22](https://github.com/twilio-labs/paste/issues/22)) ([23d7385](https://github.com/twilio-labs/paste/commit/23d7385))
-* change [@paste](https://github.com/paste) scope to [@twilio-paste](https://github.com/twilio-paste) ([#2](https://github.com/twilio-labs/paste/issues/2)) ([1d8d2ff](https://github.com/twilio-labs/paste/commit/1d8d2ff))
-* run eslint across the right files and fix any issues that arise. ([#21](https://github.com/twilio-labs/paste/issues/21)) ([2fcc872](https://github.com/twilio-labs/paste/commit/2fcc872))
-* swap light/dark border token colors ([0a99d3c](https://github.com/twilio-labs/paste/commit/0a99d3c))
-* Type-checking fixes ([#12](https://github.com/twilio-labs/paste/issues/12)) ([be02450](https://github.com/twilio-labs/paste/commit/be02450))
-
+- **design-tokens:** resolve path correctly ([6f89111](https://github.com/twilio-labs/paste/commit/6f89111))
+- **tokens:** added missing SG token values ([#22](https://github.com/twilio-labs/paste/issues/22)) ([23d7385](https://github.com/twilio-labs/paste/commit/23d7385))
+- change [@paste](https://github.com/paste) scope to [@twilio-paste](https://github.com/twilio-paste) ([#2](https://github.com/twilio-labs/paste/issues/2)) ([1d8d2ff](https://github.com/twilio-labs/paste/commit/1d8d2ff))
+- run eslint across the right files and fix any issues that arise. ([#21](https://github.com/twilio-labs/paste/issues/21)) ([2fcc872](https://github.com/twilio-labs/paste/commit/2fcc872))
+- swap light/dark border token colors ([0a99d3c](https://github.com/twilio-labs/paste/commit/0a99d3c))
+- Type-checking fixes ([#12](https://github.com/twilio-labs/paste/issues/12)) ([be02450](https://github.com/twilio-labs/paste/commit/be02450))
 
 ### Features
 
-* change the z-index scale to 10s ([#34](https://github.com/twilio-labs/paste/issues/34)) ([ed05089](https://github.com/twilio-labs/paste/commit/ed05089))
-* ComponentHeader component, component overview fixes, more ([#20](https://github.com/twilio-labs/paste/issues/20)) ([875e124](https://github.com/twilio-labs/paste/commit/875e124))
-* **website:** Add text color token accessibility rating to the token page ([#28](https://github.com/twilio-labs/paste/issues/28)) ([74f12d1](https://github.com/twilio-labs/paste/commit/74f12d1))
-* **website:** Add the tokens page ([#19](https://github.com/twilio-labs/paste/issues/19)) ([878d3c5](https://github.com/twilio-labs/paste/commit/878d3c5))
-* **website:** Callout component ([#25](https://github.com/twilio-labs/paste/issues/25)) ([e4a3d00](https://github.com/twilio-labs/paste/commit/e4a3d00))
+- change the z-index scale to 10s ([#34](https://github.com/twilio-labs/paste/issues/34)) ([ed05089](https://github.com/twilio-labs/paste/commit/ed05089))
```

Comment:
I think we should turn prettier off on changes to this file because it's auto-generated. I'd expect either diff thrashing or inconsistent markdown on future generated changelogs in this file.

Reply:
Yeah, it's a symptom of me editing it in VS code. Gunna redo these and push em back up